src/main/java/catan/services/impl/GameServiceImpl.java

```diff
@@ -293,8 +293,13 @@ private void startGame(GameBean game) {
                 return;
             }
         }
-
         log.debug("All players is ready");
+
+        log.debug("Setting resources to zero");
```

Comment:
maybe it is better to set resources when gameuser is created to reduce steps during game creation? but it just point to discuss, we don;t need t change it right now